### ISSUE TITLE
Navbar + tabs

### ISSUE THREAD

**Issue by raysummee** [Megabyte-143/certified_cliche#16]

**Comment by utkarshnashine** [comment]
i would like to work on this issue as GSSOC'22 participant.

**Comment by raysummee** [comment]
@utkarshnashine I have assigned you this issue. Please make sure you follow the folder structure. All the best.

**Comment by raysummee** [comment]
@utkarshnashine Just checking if you have any doubt? Please let us know at the discord channel.

**Comment by RaviAnand111** [comment]
Hey @raysummee  @utkarshnashine , I also want to work on this issue, if you need any help or are working on any other issue, I would like to help too.
Thanks

**Comment by raysummee** [comment]
@RaviAnand111 Thank you for your kind gesture. For now I am not allowed to assign more than one person on a single issue. 

You can work on other issues. You can find the issues on the [issue tab](https://github.com/Megabyte-143/certified_cliche/issues). You can work on any one of them which has no one assigned to. Let me know which issue you want to work on, so I assign you to that issue. If you have any doubt and question you can put it in the discord channel.

**Comment by raysummee** [comment]
@RaviAnand111 I guess @utkarshnashine is not working on it, as he's not responding to the comment. If you want to work on this let me know, I'll assign you. Thanks 😊

**Comment by RaviAnand111** [comment]
Yeah, sure @raysummee , Thanks for asking, I am definetely up for it.

**Comment by RaviAnand111** [comment]
Thanks Alot 
@raysummee

**Comment by utkarshnashine** [comment]
Can you please reassign me as it will take few more days.

**Comment by raysummee** [comment]
@utkarshnashine I have reassigned you. 

@RaviAnand111 Sorry assigning you the next issue. Thanks for your kind cooperation.
